### PR TITLE
New package: gnustep-make-2.6.8

### PR DESCRIPTION
**Pull request by asafo**

### CODE REVIEW

**Comment by asafo** [comment]
Why no gcc-obcj ??

> Installing armv7hf cross pkg: cross-vpkg-dummy ...

=> gnustep-make-2.6.8_1: building ...

   [target] gcc-objc-4.9.3_4: not found.

=> gnustep-make-2.6.8_1: installing target dependency 'gcc-objc-4.9.3_4' ...

=> ERROR: gnustep-make-2.6.8_1: failed to install 'gcc-objc-4.9.3_4' dependency! (error 2)

**Comment by Vaelatern** [comment]
Because, as I said, do not depend on that.

Please do not close and reopen PRs. It spams the developers' chatroom.

**Comment by Vaelatern** [comment]
Because, as I said, do not depend on that.

Please do not close and reopen PRs. It spams the developers' chatroom.

File: srcpkgs/gnustep-make/template [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+pkgname=gnustep-make
+version=2.6.8
+revision=1
+build_style=gnu-configure
+short_desc="GNU Step Makefile helpers for a GNUstep-based project"
+maintainer="Asaf Ohayon <asaf@sysbind.co.il>"
+license="GPL-3"
+homepage="http://www.gnustep.org"
+distfiles="http://ftp.gnustep.org/pub/gnustep/core/${pkgname}-${version}.tar.gz"
+makedepends="gcc-objc"
```
Comment (Vaelatern) [inline srcpkgs/gnustep-make/template]: bogus

**Comment by asafo** [comment]
I don't understand, it won't compile without gcc-objc,

**Comment by Vaelatern** [comment]
Also need to add `conf_files="/etc/GNUstep/GNUstep.conf"`

**Comment by Vaelatern** [comment]
https://travis-ci.org/Vaelatern/void-packages/builds/120558698 see how I don't depend on `gcc-objc`

**Pull request closed without merging**